roles/ansible-test/tasks/main.yaml

```diff
@@ -33,4 +33,13 @@
     chdir: "{{ _test_location }}"
     executable: /bin/bash
   environment: "{{ ansible_test_environment | default({}) }}"
-  shell: "source {{ ansible_test_venv_path }}/bin/activate; {{ ansible_test_executable }} {{ ansible_test_test_command }} {{ ansible_test_options }} --python {{ ansible_test_python }} -vvvv {{ _integration_targets }}"
+  shell: "source {{ ansible_test_venv_path }}/bin/activate; {{ ansible_test_executable }} network-integration {{ ansible_test_options }} --python {{ ansible_test_python }} -vvvv {{ _integration_targets }}"
```

Comment:
lets not do this, we should be able to reuse the single command for all ansible_test_test_commands, we just need to make sure it will work with both.